[2021-03-01 20:53:58,181] {taskinstance.py:826} INFO - Dependencies all met for <TaskInstance: user_processing.processing_user 2020-06-24T00:00:00+00:00 [queued]>
[2021-03-01 20:53:58,186] {taskinstance.py:826} INFO - Dependencies all met for <TaskInstance: user_processing.processing_user 2020-06-24T00:00:00+00:00 [queued]>
[2021-03-01 20:53:58,186] {taskinstance.py:1017} INFO - 
--------------------------------------------------------------------------------
[2021-03-01 20:53:58,186] {taskinstance.py:1018} INFO - Starting attempt 1 of 1
[2021-03-01 20:53:58,186] {taskinstance.py:1019} INFO - 
--------------------------------------------------------------------------------
[2021-03-01 20:53:58,196] {taskinstance.py:1038} INFO - Executing <Task(PythonOperator): processing_user> on 2020-06-24T00:00:00+00:00
[2021-03-01 20:53:58,198] {standard_task_runner.py:51} INFO - Started process 257420 to run task
[2021-03-01 20:53:58,200] {standard_task_runner.py:75} INFO - Running: ['airflow', 'tasks', 'run', 'user_processing', 'processing_user', '2020-06-24T00:00:00+00:00', '--job-id', '931', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmpqwvr_60h']
[2021-03-01 20:53:58,201] {standard_task_runner.py:76} INFO - Job 931: Subtask processing_user
[2021-03-01 20:53:58,224] {logging_mixin.py:103} INFO - Running <TaskInstance: user_processing.processing_user 2020-06-24T00:00:00+00:00 [running]> on host pop-os.localdomain
[2021-03-01 20:53:58,258] {taskinstance.py:1232} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=user_processing
AIRFLOW_CTX_TASK_ID=processing_user
AIRFLOW_CTX_EXECUTION_DATE=2020-06-24T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2020-06-24T00:00:00+00:00
[2021-03-01 20:53:58,266] {python.py:118} INFO - Done. Returned value was: None
[2021-03-01 20:53:58,268] {taskinstance.py:1142} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=processing_user, execution_date=20200624T000000, start_date=20210302T015358, end_date=20210302T015358
[2021-03-01 20:53:58,288] {taskinstance.py:1195} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2021-03-01 20:53:58,291] {local_task_job.py:118} INFO - Task exited with return code 0
